custom/icds_reports/management/commands/ap_data_pull.py

```diff
@@ -0,0 +1,52 @@
+import os
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+
+from django.core.management.base import BaseCommand
+from corehq.util.argparse_types import date_type
+from django.db import connections, transaction
+from custom.icds_reports.tasks import _get_monthly_dates
+from corehq.sql_db.connections import get_icds_ucr_citus_db_alias
+
+
+@transaction.atomic
+def _run_custom_sql_script(command, day=None):
+    db_alias = get_icds_ucr_citus_db_alias()
+    if not db_alias:
+        return
+    with connections[db_alias].cursor() as cursor:
+        cursor.execute(command, [day])
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'month',
+            type=date_type,
+            help='The start date (inclusive). format YYYY-MM-DD'
+        )
+
+    def build_data(self, date):
+        for i in range(1, 3):
+            path = os.path.join(os.path.dirname(__file__), 'sql_scripts',
+                                'ap_data_pull_{}.sql'.format(i))
+            context = {
+                'month_execute': date.strftime("%Y-%m-%d").replace("-", "_")
```

Comment:
Why are you replacing `-` with `_`? Wont it fail because in the query it would become 
`where month='2019_04_01'` instead of `where month='2019-04-01'`